x-pack/plugins/infra/public/pages/metrics/metrics_explorer/hooks/use_metrics_explorer_data.test.tsx

```diff
@@ -31,7 +35,16 @@ const renderUseMetricsExplorerDataHook = () => {
     return <KibanaContextProvider services={services}>{children}</KibanaContextProvider>;
   };
   return renderHook(
-    (props) =>
```

Comment:
Otherwise it says that `props.options` and others are missing. I didn't find a way how to nicely infer it from `useMetricsExplorerData`, so just copied types